stable/osg-frontier-squid/osg-frontier-squid/templates/configmap.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
```

```suggestion
{{ if .Values.ExtraConfig }}
apiVersion: apps/v1
```

We want to wrap the entire file in this conditional